apps/settings/src/mixins/AppManagement.js

```diff
@@ -12,16 +12,76 @@ export default {
 			return this.app.groups.map(group => { return { id: group, name: group } })
 		},
 		installing() {
+			if (this.app?.app_api) {
+				return this.app && this.$store.getters['appApiApps/loading']('install')
+			}
 			return this.$store.getters.loading('install')
 		},
 		isLoading() {
+			if (this.app?.app_api) {
+				return this.app && this.$store.getters['appApiApps/loading'](this.app.id)
+			}
 			return this.app && this.$store.getters.loading(this.app.id)
 		},
+		isInitializing() {
+			if (this.app?.app_api) {
+				return this.app && Object.hasOwn(this.app?.status, 'action') && (this.app.status.action === 'init' || this.app.status.action === 'healthcheck')
+			}
+			return false
+		},
+		isDeploying() {
+			if (this.app?.app_api) {
+				return this.app && Object.hasOwn(this.app?.status, 'action') && this.app.status.action === 'deploy'
+			}
+			return false
+		},
+		isManualInstall() {
+			if (this.app?.app_api) {
+				return this.app?.daemon?.accepts_deploy_id === 'manual-install'
+			}
+			return false
+		},
+		updateButtonText() {
+			if (this.app?.daemon?.accepts_deploy_id === 'manual-install') {
+				return t('app_api', 'manual-install apps cannot be updated')
+			}
+			return ''
+		},
 		enableButtonText() {
-			if (this.app.needsDownload) {
-				return t('settings', 'Download and enable')
+			if (this.app?.app_api) {
+				if (this.app && Object.hasOwn(this.app?.status, 'action') && this.app.status.action === 'deploy') {
+					return t('app_api', '{progress}% Deploying', { progress: this.app.status?.deploy })
+				}
+				if (this.app && Object.hasOwn(this.app?.status, 'action') && this.app.status.action === 'init') {
+					return t('app_api', '{progress}% Initializing', { progress: this.app.status?.init })
+				}
+				if (this.app && Object.hasOwn(this.app?.status, 'action') && this.app.status.action === 'healthcheck') {
+					return t('app_api', 'Healthchecking')
```

Comment:
This will cause confusion 1. by users 2. by translators, so better make it more explaining what it is (especially as this is not a normal English word).

```suggestion
					return t('app_api', 'Health checking')
```